test/link.test.js

```diff
@@ -121,80 +121,9 @@ after(function (cb) {
 
 test('put link', function (t) {
     this.client.ln(this.obj, this.key, function (err, res) {
-        t.ifError(err);
```

Comment:
likewise, do you want to remove these snaplink tests in your other change?

Reply:
ok